project.toml

```
@@ -0,0 +1,40 @@
+[build-system]
+requires = ["setuptools>=68.1.2",
+            "setuptools_scm[toml]>=6.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "GemGIS"
+authors = [
+    {name = "Alexander Jüstel", email = "alexander.juestel@rwth-aachen.de"},
+]
+description = "Spatial data processing for geomodeling"
+keywords = ["visualization", "python", "processing", "jupyter", "modeling", "geospatial", "jupyter-notebook",
+            "geographic-data", "spatial-data", "notebooks", "raster-data", "vector-data", "geographic", "geomodeling"]
+readme = "README.md"
+license = {file = LICENSE}
+dynamic = ["Version"]
+requires-python = ">=3.8"
+classifiers = [
+    'Development Status :: 5 - Production/Stable',
+    'Intended Audience :: Science/Research',
+    'Topic :: Scientific/Engineering :: Information Analysis',
+    'Programming Language :: Python :: 3.8',
+    'Programming Language :: Python :: 3.9',
+    'Programming Language :: Python :: 3.10',
+    'Programming Language :: Python :: 3.11',
+    'License :: OSI Approved :: GNU General Public License v3 (GPLv3)',
+    'Operating System :: OS Independent',
+]
+dependencies = [
+    "geopandas",
+    "rasterio",
+    "pyvista",
+]
+[project.urls]
+Documentation = 'https://gemgis.readthedocs.io/'
+Home = 'https://gemgis.readthedocs.io/'
+"Bug Tracker" = 'https://github.com/cgre-aachen/gemgis/issues'
+Repository = 'https://github.com/cgre-aachen/gemgis'
+"Source Code" = 'https://github.com/cgre-aachen/gemgis'
```

you shouldn't need the quotes here but i'm double checking on what pypa prefers. 
```suggestion
[project.urls]
Documentation = 'https://gemgis.readthedocs.io/'
Home = 'https://gemgis.readthedocs.io/'
Bug Tracker = 'https://github.com/cgre-aachen/gemgis/issues'
Repository = 'https://github.com/cgre-aachen/gemgis'
Source Code = 'https://github.com/cgre-aachen/gemgis'
```